src/dotnet/devcontainer-feature.json

```diff
@@ -28,7 +28,6 @@
         }
     },
     "containerEnv": {
-        "DOTNET_ROOT": "/usr/local/dotnet/current",
```

Comment:
Curious, won't we still need this? 

> As `DOTNET_ROOT`specifies the location of the . NET runtimes, if they are not installed in the default location. (ie /usr/local/share/dotnet for linux)

Reply:
My understanding is that we don't since the CLI itself will figure it out.

Reply:
Restored it just to avoid yet another change.  We can remove it later.